config/rbac/role.yaml

```diff
@@ -137,6 +137,14 @@ rules:
   - triggerauthentications/status
   verbs:
   - '*'
+- apiGroups:
+  - flowcontrol.apiserver.k8s.io
+  resources:
+  - prioritylevelconfigurations
+  - flowschemas
+  verbs:
+  - list
+  - watch
```

Comment:
It seems like this is the role used by the operator, is the metrics-server also using this role? Because the rbac perms don't seem to apply properly

Reply:
cluster scoped? I mean, is this for a role or cluster role?

Reply:
Currently we have a single service account using raw manifests, but for helm case we need to be care and include it to metrics server

Reply:
yeah, my mistake. Doing too many things in parallel 🤦‍♂️  will fix that